app-localize-behavior.html

```diff
@@ -194,6 +195,10 @@
     }
   },
 
+  ready() {
+    this.fire('app-localize-behavior-created', this);
```

Comment:
Maybe rename this to `app-localize-behavior-ready`? `created` is another callback, that has another meaning :)